demo/src/App.tsx

```diff
@@ -22,14 +25,19 @@ function App() {
   ];
   const [selectedLanguages, setSelectedLanguages] =
     useState<CustomScriptLanguageCode[]>(availableLanguages);
-  const [currentTime, setCurrentTime] = useState(0);
   const [isPlaying, setIsPlaying] = useState(false);
   const [volume, setVolume] = useState(0.5);
   const [playbackRate, setPlayBackRate] = useState(1);
 
-  const seekTo = (timeInSeconds: number) =>
+  const seekTo = (timeInSeconds: number) => {
+    setIsSeeking(true);
     playerRef.current?.seekTo(timeInSeconds, 'seconds');
 
+    setTimeout(() => setIsSeeking(false), 50);
+  };
```

Comment:
It looks like setIsSeeking functions similarly to the focus functionality of the focusButton. Although the focus attribute is only visible in blockview, it overlaps with setIsSeeking, so it would be better to manage them together.

Reply:
great idea!